lldb/source/Target/Target.cpp

```diff
@@ -4428,6 +4428,15 @@ void TargetProperties::SetDisableSTDIO(bool b) {
   const uint32_t idx = ePropertyDisableSTDIO;
   SetPropertyAtIndex(idx, b);
 }
+std::optional<llvm::StringRef>
+TargetProperties::GetLaunchWorkingDirectory() const {
```

Comment:
If it's not used when empty should this just return llvm::StringRef?

I get the logical purity of using optional, but if we're defining some value of it as the "null" value anyway we don't have to use optional.

Reply:
Yeah, that makes sense, also because of consistency with the rest of the code